Martyna_attempt/main.py

```diff
@@ -0,0 +1,450 @@
+import tkinter as tk
+import tkinter.ttk as ttk
+from pathlib import Path
+from yaml import load, dump, Dumper, Loader
+
+class Recipe: #1
```

Comment:
As per my comments on discord this duplicates the class that is already on main and is not require.